screenshot-test/App.screenshot.js

```diff
@@ -0,0 +1,755 @@
+    await page.waitFor(WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('Terms Page with Login'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'Terms Page with Login' });
+  })
+
+  it('FAQ Page', async() => {
+    await page.goto(`${CLIENT_URL}faq`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(LONG_WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('FAQ Page with Login'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'FAQ Page with Login' });
+  })
+
+  it('Gene Page', async() => {
+    await page.goto(`${CLIENT_URL}gene/ABL1`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(LONG_WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('Gene Page with Login'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'Gene Page with Login' });
+  })
+
+  it('Alteration Page', async() => {
+    await page.goto(`${CLIENT_URL}gene/ABL1/BCR-ABL1%20Fusion`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('Alteration Page with Login'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'Alteration Page with Login' });
+  })
+
+  it('Account Settings Page', async() =>{
+    await page.goto(`${CLIENT_URL}account/settings`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('Account Settings Page'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'Account Settings Page' });
+  })
+
+  it('Users Infomation Page', async() =>{
+    await page.goto(`${CLIENT_URL}admin/user-details`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('Users Infomation Page'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'Users Infomation Page' });
+  })
+
+  it('User Details Page', async() =>{
+    await page.goto(`${CLIENT_URL}users/admin`);
+    await page.setViewport(viewPort_1080);
+    await page.waitFor(WAITING_TIME);
+    let image = await page.screenshot(getScreenshotConfig('User Details Page'));
+    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: 'User Details Page' });
+  })
+
+  afterAll(async () => {
+    await browser.close();
+  })
+})
+
+describe('Tests without login', () => {
+  let browser;
+  let page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserConfig);
+    page = await browser.newPage();
+    await page.setRequestInterception(true); // Handle UnhandledPromiseRejectionWarning: Error: Request Interception is not enabled! 
+    page.on('request', (request) => {
+      let url = request.url()
+      if (url.includes('recaptcha')){
+        request.abort();
+      }
+      else{
+        switch (url) {
```

Comment:
Can we merge this switch with the one in the Tests with login? Are they identical?

Reply:
They are minor different, but we can merge them.